spec/controllers/selects_collections_spec.rb

```diff
@@ -7,28 +7,27 @@ class SelectsCollectionsController < ApplicationController
 end
 
 describe SelectsCollectionsController, :type => :controller do
+  let(:user) { FactoryGirl.find_or_create(:jill) }
+  let(:dit) { FactoryGirl.find_or_create(:dit) }
   describe "Select Communities" do
-    before :all do
+    before :each do
```

Comment:
* there is likely a better approach to this
 * consider function and better placement of Collection.delete_all
 * do not use before :each